route53_crossfeed_app.tf

```diff
@@ -337,6 +337,87 @@ resource "aws_route53_record" "crossfeed_integration_digicert_letsencrypt_CAA" {
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
 
+resource "aws_route53_record" "crossfeed_integration_A" {
+  provider = aws.route53resourcechange
+
+  alias {
+    name                   = "d-sl8v3ffkci.execute-api.us-east-1.amazonaws.com."
+    evaluate_target_health = false
+    zone_id                = var.api_gateway_zone_id
+  }
+  name    = "integration.crossfeed.${aws_route53_zone.cyber_dhs_gov.name}"
+  type    = "A"
+  zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
+}
```

Comment:
This resource block should be moved up to preserve alphabetical ordering by resource name in the "Integration entries" section.